test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Resolvers/OpenApiConfigurationResolverTests.cs

```diff
@@ -39,5 +39,10 @@ public void Given_Assembly_When_Resolve_Invoked_Then_It_Should_Return_Result()
 
             result.Should().BeOfType<DefaultOpenApiConfigurationOptions>();
         }
+
+        // NOTE: this abstract class is referenced via the OpenApiConfigurationResolver.Resolve method (and will be ignored)
+        public abstract class TestOpenApiConfigurationOptionsBase : DefaultOpenApiConfigurationOptions
```

Comment:
@justinyoo this validates that the abstract class is not found (and so not tried to be instantiated) by the existing test methods. If you would prefer a distinct test assembly for this test, a location for this abstract fake, or have a different test in mind, please let me know.

Reply:
Can you put this abstract class into the `Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests.Fakes` project, and put another non-abstract class so that the tests can validate both cases - throwing exceptions and not-throwing exceptions?

Reply:
I've added a base abstract and a concrete class, as well as a test to verify the types are as expected. I can't really add an exception test since the IsAbstract flag is not conditional (and the SingleOrDefault expectation in the code already shows you can only have one class per project)

Reply:
@gsteenpa Thanks for adding the test cases. That makes sense to me and verifies your changes.